core/arch/arm/plat-stm32mp1/main.c

```diff
@@ -387,25 +387,8 @@ static void configure_sysram(struct dt_driver_provider *fw_provider)
 
 static TEE_Result init_late_stm32mp1_drivers(void)
 {
-	TEE_Result res = TEE_ERROR_GENERIC;
 	uint32_t __maybe_unused state = 0;
 
-	/* Set access permission to TAM backup registers */
-	if (IS_ENABLED(CFG_STM32_TAMP)) {
-		struct stm32_bkpregs_conf conf = {
-			.nb_zone1_regs = TAMP_BKP_REGISTER_ZONE1_COUNT,
-			.nb_zone2_regs = TAMP_BKP_REGISTER_ZONE2_COUNT,
```

Comment:
Could you also remove the definition of macros `TAMP_BKP_REGISTER_ZONE*_COUNT` (*platform_config.h*) since they nor used nor relevant?